src/NetscriptFunctions.ts

```diff
@@ -1072,13 +1073,36 @@ const base: InternalAPI<NS> = {
       levelCost: Player.mults.hacknet_node_level_cost,
     };
   },
-  getBitNodeMultipliers: (ctx: NetscriptContext) => (): IBNMults => {
-    if (Player.sourceFileLvl(5) <= 0 && Player.bitNodeN !== 5) {
-      throw helpers.makeRuntimeErrorMsg(ctx, "Requires Source-File 5 to run.");
-    }
-    const copy = Object.assign({}, BitNodeMultipliers);
-    return copy;
-  },
+  getBitNodeMultipliers:
+    (ctx: NetscriptContext) =>
+    (_bitNode: unknown = Player.bitNodeN, _lvl: unknown = 1): IBNMults => {
+      const bitNode = helpers.number(ctx, "bitNode", _bitNode);
+      const lvl = helpers.number(ctx, "lvl", _lvl);
+      if (Player.sourceFileLvl(5) <= 0 && Player.bitNodeN !== 5) {
+        throw helpers.makeRuntimeErrorMsg(ctx, "Requires Source-File 5 to run.");
+      }
+
+      if (bitNode > 13 || bitNode < 1) {
+        throw helpers.makeRuntimeErrorMsg(ctx, "Invalid BitNode, can only be an integer 1-13");
+      }
+
+      if (lvl < 0 && lvl != undefined) {
+        throw helpers.makeRuntimeErrorMsg(ctx, "Invalid level, must be an integer above 0.");
+      }
+
+      if (bitNode != 12) {
+        const copy = Object.assign({}, specificBNMultipliers(bitNode, 0));
+        return copy;
+      } else if (bitNode == 12) {
```

Comment:
This section currently will always return the current multipliers if the player specifies BN12 from within BN12. e.g. if I'm on 12.4 and I'm wondering what the multipliers are going to be like at 12.8, I can't check that.

Reply:
I highlighted the wrong lines here, it should be 1096-1100